api/config-dev-exp-engine.yaml

```diff
@@ -19,15 +21,15 @@ Experiment:
         name: exp_1
         variants:
           - name: control
-          - name: treatment-1
+          - name: treatment-a
         variants_configuration:
           control:
             traffic: 0.85
             treatment_configuration:
-              foo: bar
-              route_name: treatment-a
```

Comment:
previously the treatment is `control` but it return a `route_name` of `treatment-a`, vice versa with treatment-1.

Minor refactoring to fix that and simplify the config